.github/workflows/release.yml

```diff
@@ -46,21 +50,33 @@ jobs:
           git config --local user.email "action@github.com"
           git config --local user.name "GitHub Action"
 
-      - name: maven release ${{steps.metadata.outputs.current-version}}
+      - name: Maven release ${{steps.metadata.outputs.current-version}}
         run: |
-          gpg --quiet --batch --yes --decrypt --passphrase="${{secrets.GPG_PASSPHRASE}}" --output maven-settings.xml .github/release/maven-settings.xml.gpg
           git checkout -b release
-          ./mvnw -B release:prepare -Prelease -DreleaseVersion=${{steps.metadata.outputs.current-version}} -DdevelopmentVersion=${{steps.metadata.outputs.next-version}} -s maven-settings.xml
+          ./mvnw -B release:prepare -Prelease -DreleaseVersion=${{steps.metadata.outputs.current-version}} -DdevelopmentVersion=${{steps.metadata.outputs.next-version}}
           if ! git diff --quiet docs/modules/ROOT/pages/attributes.adoc; then
             git add -f docs/modules/ROOT/pages/attributes.adoc
             git commit -m 'Update documentation version'
           fi
           git checkout ${{github.base_ref}}
           git rebase release
-          ./mvnw -B release:perform -Darguments=-DperformRelease -Prelease -s maven-settings.xml
+          ./mvnw -B release:perform -Darguments=-DperformRelease -DperformRelease -Prelease
+        env:
+          MAVEN_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+          MAVEN_PASSWORD: ${{ secrets.OSSRH_TOKEN }}
+
+      - name: Adjust tag for documentation changes
+        run: |
+          git checkout ${{steps.metadata.outputs.current-version}}
+          mvn -B clean install -DskipTests -DskipITs
+          git commit -am "Update stable version for documentation"
+          # Move the tag after inclusion of documentation adjustments
+          git tag -f ${{steps.metadata.outputs.current-version}}
+          # Go back to base ref
+          git checkout ${{github.base_ref}}
```

Comment:
What is the problem we are solving here?

Reply:
This updates the attributes.adoc in the docs module with the latest available version

Reply:
> This updates the attributes.adoc in the docs module with the latest available version

Does not this happen above around [here](https://github.com/quarkiverse/quarkus-cxf/blob/main/.github/workflows/release.yml#L54-L57)?

Reply:
No, the released version in the `attributes.adoc` is not reflected in the main branch. This step sets the released version in that file

Reply:
Actually, let me rectify that: it updates the main branch but it doesn't update the tag, just tested

Reply:
So the new `Adjust tag for documentation changes` block does the same as the existing if block plus it moves the tag.

What I do not like about this is the duplicity and the fact that we move the tag after we have performed `release:perform` using the old tag.
I wonder whether we could not remove the new `Adjust tag for documentation changes` block and just add the tag reset `git tag -f ${{steps.metadata.outputs.current-version}}` at the end of the old if block?

Reply:
AFAIK that won't work because in the `if` block the `[maven-release-plugin] prepare for next development iteration` commit already bumped to the next version, so you'll need to build without that commit and then rewrite the tag, which is what this does

Reply:
>  the [maven-release-plugin] prepare for next development iteration commit already bumped to the next version

Indeed, thanks explaining the obvious :) As long as I'll see some potential for improvements, I'll do it in a followup.